run.sh

```diff
@@ -21,20 +21,51 @@
 
 # Runs Polaris as a mini-deployment locally. Creates two pods that bind themselves to port 8181.
 
-CURRENT_DIR=$(pwd)
+# Initialize variables
+ECLIPSELINK="false"  # Default value
 
-# deploy the registry
+# Function to display usage information
+usage() {
+  echo "Usage: $0 [-e true|false] [-h]"
+  echo "  -e    Set the ECLIPSELINK flag (default: false)"
+  echo "  -h    Display this help message"
+  exit 1
+}
```

Comment:
This change is one I'm a little unsure about. Things like metastore manager and callcontext resolver are designed to be pluggable and discoverable by DropWizard. 

I'm worried about the precedent this sets and whether we would want to add all these as myriad options here. The script might get quite complicated in order to cover every combination.

Reply:
With this concern, should we move those into a config file instead?

Reply:
Or maybe just a safety valve type flag that allows you to pass an arbitrary build arg down to `docker build`?

Reply:
K, let me think about it tonight then submit another PR to address those tonight.

Reply:
@eric-maynard updated with arbitrary build arg support. Please take another look when you get a chance.

Reply:
actually lets have it in a diff PR. The current Dockerfile doesn't support eclipseLinkDeps flag. I will add this in and update the doc for both optional options. What do u think?

@flyrain

Reply:
I'd recommend to add an example like  `./run.sh -b ECLIPSELINK=true`, as it is probably most commonly used.